universal-application-tool-0.0.1/app/views/components/SelectWithLabel.java

```diff
@@ -72,6 +72,11 @@ public SelectWithLabel setDisabled(boolean disabled) {
 
   @Override
   public ContainerTag getContainer() {
+    Tag placeholder = option(placeholderText).withValue("").attr(Attr.HIDDEN);
+    if (this.fieldValue.isEmpty()) {
+      placeholder.attr(Attr.SELECTED);
```

Comment:
Question: what does it mean when a hidden element is selected?

Reply:
It behaves as if it's selected, but when you open the dropdown it's hidden in the list itself.  So that combination makes it show in the form initially but keeps the user from being able to select it deliberately.